### ISSUE TITLE
UML

### ISSUE THREAD

**Issue by r-gaia-cs** [ufpr-opt/perprof-jors#16]
Tyr http://www.logilab.org/blogentry/6883 for UML.

**Comment by r-gaia-cs** [comment]
1. Install `graphviz`:

   ~~~
   # apt-get install graphviz
   ~~~

2. Install `Pylint`:

   ~~~
   # pip install pylint
   ~~~

3. Run

   ~~~
   $ pyreverse -o png -p perprof perprof
   ~~~

We will get 
![classes_perprof](https://cloud.githubusercontent.com/assets/1506457/7870272/99935d3a-055e-11e5-8c14-b096600e7889.png)

**Comment by abelsiqueira** [comment]
You forgot `-a`.

**Comment by r-gaia-cs** [comment]
Addressed by 8097631
